lecture_1/homework/app/controllers/application_controller.rb

```diff
@@ -1,2 +1,9 @@
+# frozen_string_literal: true
+require("Errorable.rb")
```

Comment:
Zazwyczaj jak w railsach musimy pisać require to znaczy, że coś umieściliśmy w niewłaściwym miejscu lub nadaliśmy niewłaściwą nazwę. Ale w tym przypadku wszystko wygląda ok po usunięciu tego require.

Zwróć natomiast uwagę na to jak zwykle nazywane są pliki. W nazwach klas używamy `CammelCase`, ale w nazwach plików już `snake_case`

Reply:
Ach, to chyba problem nie-ładowania plików, o którym było na ostatnim wykładzie. ;) 

Dzięki za uwagę!